### DISCUSSION TITLE
get elevation error in gis addon

### DISCUSSION THREAD

**Discussion by hassanhn** [domlysz/BlenderGIS#630] [Q&A]
hello 
I import normally basemap
but when we import elevation to get 3d model i have a issue :cannot reach opentopography web service
how can i solve this problem

**Comment by lobnico** [comment]
Hello, I made a PR, 
you will need to get an api key from opentopography website --just register and you will be prompted to request it

Since author doesn t respond to PR, use my branch (https://github.com/lobnico/BlenderGIS) 
then in blender addon settings put your api key.

**Comment by hassanhn** [comment]
Hello dear I register a account in opentopography but I don't know what
next to do

**Comment by domlysz** [comment]
Fixed https://github.com/domlysz/BlenderGIS/commit/72cbb0507b80c2116fd4252b9082b6beba170a89
